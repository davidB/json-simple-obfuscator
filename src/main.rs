pub fn main() {}

#[cfg(test)]
mod tests {
    use indoc::indoc;

    #[test]
    fn test_sample() {
        insta::assert_snapshot!(indoc! {r#"
                {
                    "a": "Hello",
                    "id": 123456,
                    "details": {
                        "user": "johnD",
                        "name": "John Doe",
                        "url": "http://example.com/item/123456"
                    }
                }
            "#},);
    }
}
