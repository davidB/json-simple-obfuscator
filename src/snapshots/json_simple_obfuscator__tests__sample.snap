---
source: src/main.rs
expression: "indoc!\n{\n    r#\"\n                {\n                    \"a\": \"Hello\",\n                    \"id\": 123456,\n                    \"details\": {\n                        \"user\": \"johnD\",\n                        \"name\": \"John Doe\",\n                        \"url\": \"http://example.com/item/123456\"\n                    }\n                }\n            \"#\n}"
---
{
    "a": "Hello",
    "id": 123456,
    "details": {
        "user": "johnD",
        "name": "John Doe",
        "url": "http://example.com/item/123456"
    }
}
